deployment/v3/mosip/prereg/install.sh

```
@@ -9,25 +9,33 @@ fi
   API_HOST=`kubectl get cm global -o jsonpath={.data.mosip-api-host}`
   PREREG_HOST=`kubectl get cm global -o jsonpath={.data.mosip-prereg-host}`
-
+  
+  # Prefer YAML host if defined, else fallback
+  FINAL_PREREG_HOST=${PREREG_GATEWAY_HOST:-$PREREG_HOST}
   echo Install prereg-gateway
-  helm -n $NS install prereg-gateway mosip/prereg-gateway --set istio.hosts[0]=$PREREG_HOST --version $CHART_VERSION
+  helm -n $NS install prereg-gateway mosip/istio-addons \
+    --set istio.name=prereg-gateway \
+    --set istio.ingressController=ingressgateway \
+    --set istio.host=$FINAL_PREREG_HOST \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)